### ISSUE TITLE
PEP 541 Request: win32gui

### ISSUE THREAD

**Issue by mhammond** [pypi/support#3852]
### Project to be claimed

`win32gui`: https://pypi.org/project/win32gui

### Your PyPI username

`mhammond`: https://pypi.org/user/mhammond

### Reasons for the request

This project is confusing some users who are actually looking for pywin32 - eg, https://github.com/mhammond/pywin32/issues/2239. While I'm listed as the "author" of the page, it's actually owned by a ghost (which I assume means a deleted user?)

Ultimately I'd like this project deleted, but if transferring it to me is the best way of achieving that, please transfer it.

### Maintenance or replacement?

Replacement

### Source code repositories URLs

https://github.com/mhammond/pywin32

### Contact and additional research

I don't know how to attempt to contact https://pypi.org/user/ghost555/

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by Thespi-Brain** [comment]
Hi @mhammond,

Do you happen to know why the owner of this project may use your email as the contact? Just wanted to make sure your account wasn't compromised or anything.

**Comment by mhammond** [comment]
That project has `ghost555` as the maintainer - my name is there, but it looks like it came from the setup.py metadata. I can't see any evidence someone has managed to use my account here?

**Comment by Thespi-Brain** [comment]
Hello @mhammond,

The user is an actual user, here, ghost does not mean "ghost" as in a deleted user. Are you still interested in getting this name? It has been almost two years.

**Comment by mhammond** [comment]
> Hello [@mhammond](https://github.com/mhammond),
> 
> The user is an actual user, here, ghost does not mean "ghost" as in a deleted user. Are you still interested in getting this name? It has been almost two years.

Would that help? win32gui is part of pywin32, there should be no pypi project of that name in the interests of not confusing users.

**Comment by Thespi-Brain** [comment]
No, this would neither help nor hurt. You listed only one instance of a user somehow confusing `pywin32` for `win32`. If you have a significant number of users who are having this problem, (which doesn't seem to be the case since there is a clear difference between `pywin32` and `win32` and if the names were actually similar, PyPI's ultranormalization rules would have prohibited one or the other projects from forming), then we could look into next steps. It's been two years so I am closing this issue.
